### ISSUE TITLE
`--compressed` breaks `curl` calls

### ISSUE THREAD

**Issue by pheerai** [msys2/MSYS2-packages#1579]
When issuing a command like `curl --compressed https://github.com`, I get the following error:

```
curl: (43) A libcurl function was given a bad argument
```

This does not occur when excluding the `--compressed`-option. On first glance, it looks as if there's some missmatch beetween `curl` and `libcurl`.

# Versions used

Curl (`curl --version`):
```
curl 7.64.0 (x86_64-pc-msys) libcurl/7.64.0 OpenSSL/1.1.1a zlib/1.2.11 brotli/1.0.7 libidn2/2.1.0 libpsl/0.20.2 (+libidn2/2.1.0) libssh2/1.8.0 nghttp2/1.36.0
Release-Date: 2019-02-06
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: AsynchDNS Debug TrackMemory IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz brotli TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL Metalink
```

curl (excerpt from `pacman -Qi curl`):
```
Version         : 7.64.0-1
Architecture    : x86_64
Build Date      : Thu, Feb 7, 2019 10:48:32
Install Date    : Wed, Feb 20, 2019 10:36:46
```

libcurl (excerpt from `pacman -Qi libcurl`):
```
Version         : 7.64.0-1
Architecture    : x86_64
Build Date      : Thu, Feb 7, 2019 10:48:32
Install Date    : Wed, Feb 20, 2019 10:36:45
```

msys2-runtime (excerpt from `pacman -Qi msys2-runtime`):
```
Version         : 2.11.2-1
Architecture    : x86_64
Build Date      : Mon, Nov 26, 2018 07:08:56
Install Date    : Tue, Nov 27, 2018 08:04:04
```

**Comment by cbrt64** [comment]
Currently works for me. Curl was upgraded a week later to 7.64.0-2 (bcc6e41) to fix a library incompatibility, we're at 7.64.1-1 as of four days ago.

**Comment by pheerai** [comment]
7.64.1-1 fixed the issue for me.
Thanks!
